src/sentry/runner/commands/storeconsumer.py

```diff
@@ -0,0 +1,50 @@
+from __future__ import absolute_import, print_function
+
+import click
+import signal
+from django.conf import settings
+
+from sentry.runner.decorators import configuration
+
+
+@click.command()
```

Comment:
The following (and the batching consumer) comes from the tried-and-true Snuba consumer, fwiw. Not a lot going on here.